main.js

```diff
@@ -67,4 +79,8 @@ const menubar = Menubar({
   ipcMain.on('app-ready', event => {
     event.sender.send('port-ready', port);
   });
+
+  ipcMain.on('error', function(e){	
```

Comment:
Can you use arrow function here?

```suggestion
  ipcMain.on('error', event =>
```